### ISSUE TITLE
plex monitor sometimes stops reporting

### ISSUE THREAD

**Issue by saqebakhter** [hippojay/plugin.video.plexbmc#134]
I've noticed with plexWatch that sometimes clients that use plexbmc - even though they watch the full movie/episode the watched % is not reported correctly.

It looks like the plexmonitor function stops sending progress updates to plex - i'm unsure of why it happens though. 

Hoping you can look into it.

**Comment by hippojay** [comment]
Thanks. I don't use plexwatch myself, but I've not come across any issues so far with mis reporting of the watch status on the clients.  

When this happens, are you watching to completion or do you stop the media early (for example when the credits start)?

**Comment by saqebakhter** [comment]
Both I'm unable to find a pattern on when it fails to report....

**Comment by hippojay** [comment]
Ok.  The monitoring of the watched status occurs every couple of seconds, so even if one attempt fails enough tries should occur as to not miss it.  It does rely on KODI to tell me the playback time, and perhaps it is this failing (as this would stop all reporting for that media file).  This would be caught and logged, so if it does happen again, if you could get the kodi.log file that might help to pin it

**Comment by Monstar316** [comment]
I had the exact same issue yesterday when watching an entire movie. I use PlexPy to monitor Plex usage and PlexBMC has been working great with Kodi until yesterday where PlexPy was reporting that I only viewed 13 minutes of the movie when I actually watched the full thing. Plex itself was showing that I had only partially watched the movie as did Kodi.
Here is my Kodi.log file. I've only taking it from when PlexBMC started to roughly the time the reporting stopped.

[Kodi.txt](https://github.com/hippojay/plugin.video.plexbmc/files/75489/Kodi.txt)
